portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -81,12 +123,55 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/comments.html");
 
   }
+
+  /* 
+    Converts an ArrayList into a Json format
+    */
   public String convertJson(ArrayList<String> messages) throws IOException {
     Gson gson = new Gson();
     String json = gson.toJson(messages);
     System.out.println(json);
     return json;
-    //response.setContentType("text/html;");
-    //response.getWriter().println("Hi! I'm Jennifer,");
+
+  }
+  
+    /*
+        Retrieves an url from BlobStore that would
```

Comment:
For multiline comments, please start each line with a *.

https://google.github.io/styleguide/javaguide.html#s4.8.6-comments

Reply:
Alright!